app/components/add-reminder.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  store: Ember.inject.service(),
+
+  tagName: 'section',
+  classNames: ['add-reminder'],
+
+  text: '',
+
+  actions: {
+    createReminder() {
```

Comment:
This issue asks simply for all reminders to be shown on '/'. Remove additional code until a later issue.